packages/farms/src/fetchFarmsV3.ts

```diff
@@ -44,20 +44,22 @@ export async function farmV3FetchFarms({
         return null
       }
       const lmPoolAddress = v3PoolData[index][1][0]
+      const allocPoint = poolInfos[index]?.allocPoint
```

Comment:
do we still need allocPoint if we have poolWeight?

Reply:
fixed